ERCS/erc-7741.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 7741
+title: Authorize Operator
+description: Set Operator via EIP-712 secp256k1 signatures
+author: Jeroen Offerijns (@hieronx), João Martins (@0xTimepunk)
+discussions-to: https://ethereum-magicians.org/t/erc-7741-authorize-operator/20531
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-06-03
+requires: 712, 1271
+---
+
+## Abstract
+
+A set of functions to enable meta-transactions and atomic interactions with contracts implementing an operator model, via signatures conforming to the [EIP-712](./eip-712.md) typed message signing specification.
```

Comment:
Again, could probably do with a definition of "operator" here. I assume you mean a role/permission for a contract, and not something like the "unary negation" operator :rofl: